scripts/authorize-metrics.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# (c) Copyright [2021-2022] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script that will authorize the given service account in the given namespace
+# the ability to access a protected metrics endpoint.
+
+set -o errexit
+set -o pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+REPO_DIR=$(dirname $SCRIPT_DIR)
+OP_SA=verticadb-operator-controller-manager
```

Comment:
In case of custom sa what do we do?

Reply:
I use this in a single test currently, which does not provide its own ServiceAccount.  So, the default is fine.  I will update the script to add an option to override this in case we ever use this in other places.